storm-server/src/main/java/org/apache/storm/DaemonConfig.java

```diff
@@ -74,6 +74,12 @@
     @IsString
     public static final String STORM_DAEMON_METRICS_REPORTER_PLUGIN_DOMAIN = "storm.daemon.metrics.reporter.plugin.domain";
 
+    /**
+     * We report the metrics with this interval period.
+     */
+    @IsString
+    public static final String STORM_DAEMON_METRICS_REPORTER_INTERVAL_PERIOD = "storm.daemon.metrics.reporter.interval.period";
```

Comment:
Please rename so we include the time unit, e.g. "storm.daemon.metrics.reporter.interval.secs"

Reply:
sure, will do